web/src/app/enterprise/sponsors/page.tsx

```
@@ -0,0 +1,427 @@
+            </div>
+            <p className="text-gray-400 text-sm leading-relaxed group-hover:text-gray-300 transition-colors duration-300">{tier.description}</p>
+          </div>
+
+          <div className="flex-1">
+            <div className="space-y-3">
+              {tier.benefits.map((benefit, i) => {
+                const isStatement = benefit === 'Let\'s build something together.'
+                return (
+                  <div key={i} className={`flex items-start gap-2 ${isStatement ? 'mt-4 pt-4 border-t border-[#30363d]' : ''}`}>
+                    {!isStatement && (
+                      <svg className="w-5 h-5 text-[#fd7e14] shrink-0 mt-0.5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)